src/utils/export.js

```diff
@@ -0,0 +1,10 @@
+import { INDICATOR_THRESHOLD_TRANSFORMERS } from 'constants/indicators';
+
+export const getExportSql = (indicator, thresholdParam, points = []) => {
+  const threshold = INDICATOR_THRESHOLD_TRANSFORMERS[indicator](thresholdParam) || 0;
+  const rawField = indicator.replace('cat', 'raw');
+  const labelField = indicator.replace('cat', 'label');
+  const pointsSql = points.map(({ lat, lng }) => `st_within(CDB_LatLng(${lat}, ${lng}), r.the_geom)`).join(' or ');
```

Comment:
Nice! This might be too slow when a lot of points are selected, but we can cross that bridge if it's an issue.